test/functional/tests/cache_ops/test_concurrent_cores_flush.py

```diff
@@ -0,0 +1,102 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+from time import sleep
+import pytest
+
+from api.cas import casadm, casadm_parser, cli
+from api.cas.cache_config import CacheMode, CleaningPolicy, CacheModeTrait
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from core.test_run import TestRun
+from test_tools.dd import Dd
+from test_utils.output import CmdException
+from test_utils.size import Size, Unit
+
+cache_size = Size(4, Unit.GibiByte)
+
+
+@pytest.mark.parametrize("cache_mode", CacheMode.with_traits(CacheModeTrait.LazyWrites))
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_concurrent_cores_flush(cache_mode):
+    """
+        title: Test if OpenCAS works correctly when concurrent cores makes flushing simultaneously.
+        description: |
+          Concurrent cores of shared cache should be able to flush dirty data simultaneously
+          and independently.
```

Comment:
I think description and title are misleading. `Fail to flush two cores simultaneously` as title and `CAS should return an error on attempt to flush second core if there is already one flush in progress` as description are more precise imo. After reading current version I thought you are testing `casadm -F` command, not `casadm -E` called twice.